src/content/blog/2025-10-23-redesigned-publisher-tools-guide.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'How the Redesigned Publisher Tools Work: A Technical Guide'
+description: 'A technical exploration of the architecture details behind the redesigned Publisher Tools'
+date: 2025-10-23
+slug: redesigned-publisher-tools-guide
+authors:
+  - Darian Avasan
+author_urls:
+  - https://darianm.xyz/
```

Comment:
This looks like a Web Monetization test site. Usually people link to personal portfolios or LinkedIn.